Makefile

```diff
@@ -124,7 +135,10 @@ coverage:
 COVER_REPORT_OPTS ?= -select_re ^lib/
 
 .PHONY: coverage-codecov
-coverage-codecov: coverage
+coverage-codecov: coverage coverage-report-codecov
```

Comment:
careful as this could be triggered in parallel now (in theory). One can specify `.NOTPARALLEL: …` but I think the report really depends on `coverage`.

Reply:
Ok, since we can use `cover report` commands directly in CI config - it is better to not remove possibility of parallel run (these targets were not changed)